superset/assets/src/visualizations/deckgl/layers/arc.jsx

```diff
@@ -18,7 +18,8 @@ function getPoints(data) {
   return points;
 }
 
-function getLayer(fd, data, slice) {
+function getLayer(fd, payload, slice) {
```

Comment:
Good catch, I simplified the `getLayer` interface but forgot that other deck.gl vizs need other attributes in the payload.